### PR TITLE
chore(suite): better percentrage formatting

### PR DESCRIPTION
**Pull request by peter-sanderson**
## Description

User localize function for percents

## Related Issue

Resolve

### CODE REVIEW

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx [hunk 1]
```diff
@@ -29,9 +33,8 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
     return (
         <LogoWrapper>
             <Tooltip
-                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
+                content={`${localizeNumber(assetPercentage ?? 0, locale, 2, 2)}%`}
```
Comment (dahaca) [inline packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: nit: if you have time would be sweet to have a util `localizePercentage(assetPercentage, locale)`

**Review by dahaca — APPROVED** [review]

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: or use Intl as we discussed – great idea!

Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: https://github.com/trezor/trezor-suite/pull/10973/commits/5cb05f59d9fe3ecdf48c945434b5424126ac4562

File: suite-common/wallet-utils/src/localizePercentage.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export type FormatPercentageArgs = {
+    value: number;
+    locale: string;
+    numDecimals: number;
+};
+
+/**
+ * Format number between 0 and 1 to percentage locale string.
+ */
+export const localizePercentage = ({ value, locale, numDecimals }: FormatPercentageArgs) => {
```
Comment (dahaca) [inline suite-common/wallet-utils/src/localizePercentage.ts]: nit: how about this, just to be more explicit? Not sure what the default should be though :D 

```suggestion
export const localizePercentage = ({ value, locale, numDecimals = 1 }: FormatPercentageArgs) => {
```

File: suite-common/wallet-utils/src/localizeNumberUtils.ts [hunk 3]
```diff
@@ -23,15 +23,15 @@ export const localizeNumber = (
     const { decimalSeparator, thousandsSeparator } = getLocaleSeparators(locale);
 
     const getDecimalsLength = () => {
-        const originalDecimalsLegth = amount.decimalPlaces() ?? 0;
-        if (originalDecimalsLegth < minDecimals) {
+        const originalDecimalsLength = amount.decimalPlaces() ?? 0;
+        if (originalDecimalsLength < minDecimals) {
```
Comment (dahaca) [inline suite-common/wallet-utils/src/localizeNumberUtils.ts:27]: 👍

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx [hunk 4]
```diff
@@ -29,9 +33,12 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
     return (
         <LogoWrapper>
             <Tooltip
-                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
+                content={localizePercentage({
+                    value: assetPercentage ?? 0,
+                    locale,
+                    numDecimals: 2,
+                })}
```
Comment (dahaca) [inline packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: nit: how about also chageing it in a couple of other places, [here](https://github.com/trezor/trezor-suite/blob/a513f54f8c10fb8d008170397a7cd647ac2ff125/packages/suite/src/components/suite/Ticker/TrendTicker.tsx#L56) for example?

**Review by dahaca — APPROVED** [review]
Neat addition! 

We also have a `formatters` package actually! How about adding a percentage formatter there some time later? @Nodonisko how are you dealing with it on mobile? 👀

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx (hunk 4 above)
Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: :+1:

**Pull request merged**